tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -1046,6 +964,7 @@
                 </sourcefiles>
             </structure>
             <html destdir="${code-coverage.dir}"/>
+            <xml destfile="${code-coverage.dir}/report.xml" />
```

Comment:
Consider naming this file something more descriptive, such as `coverage-result.xml`, to make it clear what the file contains if someone downloads the artifact to their local machine.

Reply:
The xml coverage result is the report supported by the github actions (both the actions I've tried), it won't be uploaded as artifact, but I can rename it regardless. Also, this file is created inside the code-coverage folder.